governing-documents/project-annual-review.md

```diff
@@ -0,0 +1,92 @@
+* At the conclusion of the public meeting, the TOC votes to approve the annual review. Should a concern be registered on a project, the vote will be held separately.
+* After the meeting wraps up, the assigned TOC member should summarize the discussion on the PR in the form of a comment to document information for the project and community.
+
+### Filing an Annual Review
+Hyperledger Foundation staff will notify the project maintainers and copy the TOC when the project review is due. 
+
+Project maintainers are responsible for agreeing between them who will complete the annual review. One of the maintainers should create the review in GitHub under [hyperledger-foundation/toc/project-updates/](https://github.com/hyperledger-foundation/toc/tree/main/project-updates). 
+
+* Raise a PR titled *[Project name] [year] Annual Review* (e.g., Amazing Project 2024 Annual Review)
+* The PR should include a file called `./<year>/<year>-<project name>-annual.md` (e.g., `2024/2024-amazingproj-annual.md`) with the [contents described below](#annual-review-contents)
+* Send an email to the [TOC mailing list](mailto:toc@lists.hyperledger.org) so that the community knows the PR is there and can comment on it.
+
+**If your annual review is not submitted within two months of notification, we will take this as a sign that the project is not under active maintenance and the TOC is likely to decide to archive the project and move it to End of Life status.**
+
+**NOTE:** If a project has genuinely stalled we can save everyone’s time and effort by archiving it.
+
+### Annual Review Contents
+Your annual review should answer the following questions: 
+
+* Include a link to your project’s [LFX Insights page](https://insights-v2.lfx.linuxfoundation.org/projects). We will be looking for signs of consistent or increasing contribution activity. Please feel free to add commentary to add color to the numbers and graphs we will see on Insights.
+* How many maintainers do you have, and which organisations are they from? (Feel free to link to an existing [MAINTAINERS file](../guidelines/MAINTAINERS-guidelines.md) if appropriate.)
+* What do you know about adoption, and how has this changed since your last review or since being accepted into Hyperledger Foundation? If you can list companies that are adopters of your project, please do so. Feel free to link to an existing ADOPTERS file if appropriate.
+* How has the project performed against its goals since the last review? (We won't penalize you if your goals changed for good reasons.)
+* What are the goals for the next year of the project? For example, are you working on major new features? Or are you concentrating on adoption or documentation? 
+* How can the Hyperledger Foundation help you achieve your upcoming goals? 
+* Do you think that your project meets the criteria for another stage (see [Project Lifecycle](./project-lifecycle.md))?
```

Comment:
another stage --> next stage?

Reply:
As discussed in the TOC meeting, we will leave this as is given potential changes that might come out of the project lifecycle/badging task force.